testing/MLDBFB-192_row_name_as_string_test.py

```diff
@@ -0,0 +1,47 @@
+#
+# MLDBFB-192_row_name_as_string_test.py
+# Mich, 2016-02-02
+# Copyright (c) 2016 Datacratic Inc. All rights reserved.
+#
+import tempfile
+import os
+import unittest
+mldb = mldb_wrapper.wrap(mldb) # noqa
+
+class RowNameAsStringTest(MldbUnitTest):
+    query = "SELECT * FROM csv ORDER BY rowName()"
+
+    @classmethod
+    def setUpClass(cls):
+        tmp_file = tempfile.NamedTemporaryFile(
+            prefix=os.getcwd() + '/build/x86_64/tmp/')
```

Comment:
This will keep adding new files in tmp as we rerun the test.  Consider using a fixed name containing the JIRA number.